src/tox/tox_env/package.py

```diff
@@ -97,7 +97,8 @@ def mark_active_run_env(self, run_env: RunToxEnv) -> None:
         self._envs.add(run_env.conf.name)
 
     def teardown_env(self, conf: EnvConfigSet) -> None:
-        self._envs.remove(conf.name)
+        if conf.name in self._envs:
```

Comment:
Is that an acceptable workaround @gaborbernat?

Otherwise this throws an exception since `conf.name` (`".tox"`) is not found in `self._envs` when tearing down the automatically provisioning environment.